jsonschema/_validators.py

```diff
@@ -377,3 +377,36 @@ def not_draft4(validator, not_schema, instance, schema):
         yield ValidationError(
             "%r is not allowed for %r" % (not_schema, instance)
         )
+
+
+def required_with_list(validator, required, instance, schema):
+    if not validator.is_type(instance, "object"):
+        return
+    for property in required:
```

Comment:
Só a título de curiosidade mesmo... essa palavra `property` 'eu uma palavra reservada do python, ela ta sendo usada como iterador dentro desse `for` por que? qual o valor que ela retorna?

Reply:
Só uma variável com nome property msm.
Deixei assim pra manter o padrão da lib que só usa assim.
Mas posso mudar todas as ocorrências de "property" pra "prop" se for o caso.

Reply:
Alterei